frontend_v2/README.md

```diff
@@ -0,0 +1,53 @@
+# Step-by-step coding of the frontend
+
+Are you new to JavaScript, React and Pixi?  
+Do you want to contribute or simply understand the code powering the visual frontend of `tqecjs`?  
+Look no further.  
+
+In this folder we will re-build the frontend adding a few lines of codes at a time.
+The commit message will make clear which functionality is added (or, at least, the goal is to have informative commits).
```

Comment:
The 50 or so commits you will merge will be squashed into one. You could provide a link to this PR in this readme, which contains the full commit log.

Reply:
Ok.